theoplayer/how-to-guides/android/media3/getting-started.mdx

```diff
@@ -0,0 +1,99 @@
+# Getting started with Media3 on Android
+
+<Intro />
+
+## Usage
+
+1. Follow [our Getting Started guide](../../../getting-started/01-sdks/02-android/00-getting-started.mdx)
+   to set up THEOplayer in your Android app.
+2. Add the Media3 Integration as a dependency in your module-level `build.gradle` file:
+3. Add the Media3 Integration to the player.
+
+### Add the Media3 Integration dependency
```

Comment:
Nit-pick: write "integration" in lowercase.
```suggestion
### Add the Media3 integration dependency
```
(This occurs a couple of times in this file.)